integration/integration-4/datarepo-api.yaml

```diff
@@ -13,6 +13,9 @@ env:
   IT_JADE_API_URL: https://jade-4.datarepo-integration.broadinstitute.org
   AZURE_CREDENTIALS_APPLICATIONID: 22cb243c-f1a5-43d8-8f12-6566bcce6542
   AZURE_CREDENTIALS_HOMETENANTID: fad90753-2022-4456-9b0a-c7e5b934e408
+  RBS_ENABLED: true
+  RBS_POOL_ID: datarepo_v1
+  RBS_INSTANCE_URL: https://buffer.tools.integ.envs.broadinstitute.org
```

Comment:
int 4 and 5 shouldn't use tools since objects in there are persistent

Reply:
oh... Will update!